class_lib/linear_regression.py

```diff
@@ -0,0 +1,119 @@
+from sklearn.datasets import load_boston
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import LinearRegression
+from sklearn.metrics import mean_absolute_error
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pickle
+import os, sys
+sys.path.insert(1, os.path.abspath('.'))
+
+from config import *
+
+
+#print(OUTPUT_PATH)
+
+class Predict_Price:
```

Comment:
I appreciate the effort of trying to do this in an OOP way. But honestly, I don't think it was necessary here.
Model training is not part of API repo. Train a model in a notebook, evaluate it, save it, and just have code for loading/using it here.

Reply:
Acknowledged